common/lib/capa/capa/inputtypes.py

```diff
@@ -525,9 +526,10 @@ def extract_choices(element, i18n):
                 choices.append((choice.get("name"), stringify_children(choice)))
             else:
                 if choice.tag != 'compoundhint':
-                    msg = u'[capa.inputtypes.extract_choices] {error_message}'.format(
+                    msg = Text('[capa.inputtypes.extract_choices] {error_message}').format(
                         # Translators: '<choice>' and '<compoundhint>' are tag names and should not be translated.
-                        error_message=_('Expected a <choice> or <compoundhint> tag; got {given_tag} instead').format(
+                        error_message=Text(
+                            _('Expected a <choice> or <compoundhint> tag; got {given_tag} instead')).format(
```

Comment:
Definitely Text. And since Text is the default, you don't actually have to specify Text on line 533.